src/components/Greetings/Greetings.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Greetings.css';
+
+function Greetings(props) {
```

Comment:
In this kind of exercises when the questions is something like when you have this, do that. The switch statement is always the best aproach. Nevertheless, good job!